go/vt/vtorc/logic/keyspace_shard_discovery.go

```diff
@@ -28,6 +28,19 @@ import (
 	"vitess.io/vitess/go/vt/vtorc/inst"
 )
 
+var (
+	// keyspaceShardNames stores the current names of shards by keyspace.
+	keyspaceShardNames   map[string][]string
+	keyspaceShardNamesMu sync.Mutex
+)
+
+// GetKeyspaceShards returns the shard records for a keyspace.
+func GetKeyspaceShards(keyspaceName string) []string {
+	keyspaceShardNamesMu.Lock()
+	defer keyspaceShardNamesMu.Unlock()
+	return keyspaceShardNames[keyspaceName]
+}
+
 // RefreshAllKeyspacesAndShards reloads the keyspace and shard information for the keyspaces that vtorc is concerned with.
 func RefreshAllKeyspacesAndShards() {
```

Comment:
This func is called by a ticker-loop with a duration == to `--topo-information-refresh-duration` _(in `vtorc.go`)_